docs/docs/outputs/output-formats.md

```diff
@@ -14,65 +13,53 @@ The following output formats are supported:
 - `webhook:http://url/webhook` - send events in json format to the webhook url
 - `none` - ignore stream of events output, usually used with --capture
 
+Note that only one output format can be used in the Tracee configuration.
+
 ## Examples
 
-### Table
+The following examples will have to be added into a Tracee configuration file.
 
-```console
-sudo ./dist/tracee --output table --scope comm=bash --scope follow --events openat
-```
+### Table
 
-```text
-TIME             UID    COMM             PID     TID     RET              EVENT                ARGS
-11:21:51:254199  1000   exa              1639459 1639459 3                openat               dirfd: -100, pathname: /etc/ld.so.cache, flags: O_RDONLY|O_CLOEXEC, mode: 0
-11:21:51:254285  1000   exa              1639459 1639459 3                openat               dirfd: -100, pathname: /lib/x86_64-linux-gnu/libgcc_s.so.1, flags: O_RDONLY|O_CLOEXEC, mode: 0
-11:21:51:254418  1000   exa              1639459 1639459 3                openat               dirfd: -100, pathname: /lib/x86_64-linux-gnu/libm.so.6, flags: O_RDONLY|O_CLOEXEC, mode: 0
+In the CLI:
 
-End of events stream
-Stats: {EventCount:3 EventsFiltered:0 NetCapCount:0 BPFLogsCount:0 ErrorCount:0 LostEvCount:0 LostWrCount:0 LostNtCapCount:0 LostBPFLogsCount:0}
+```yaml
+output:
+    table:
+        files:
+            - /path/to/table1.out
+            - /path/to/table2.out
 ```
 
 ### Table (Verbose)
 
-    ```console
-    sudo ./dist/tracee --output table-verbose --scope comm=bash --scope follow --events openat
-    ```
+In the CLI:
 
-    ```text
-    TIME             UTS_NAME         CONTAINER_ID  MNT_NS       PID_NS       UID    COMM             PID     TID     PPID    RET              EVENT                ARGS
-    11:22:16:970700  fujitsu                        4026531840   4026531836   1000   exa              1643836 1643836 3795408 3                openat               dirfd: -100, pathname: /etc/ld.so.cache, flags: 524288, mode: 0
-    11:22:16:970783  fujitsu                        4026531840   4026531836   1000   exa              1643836 1643836 3795408 3                openat               dirfd: -100, pathname: /lib/x86_64-linux-gnu/libgcc_s.so.1, flags: 524288, mode: 0
-    11:22:16:970913  fujitsu                        4026531840   4026531836   1000   exa              1643836 1643836 3795408 3                openat               dirfd: -100, pathname: /lib/x86_64-linux-gnu/libm.so.6, flags: 524288, mode: 0
-    
-    End of events stream
-    Stats: {EventCount:3 EventsFiltered:0 NetCapCount:0 BPFLogsCount:0 ErrorCount:0 LostEvCount:0 LostWrCount:0 LostNtCapCount:0 LostBPFLogsCount:0}
-    ```
+```yaml
+output:
+    table-verbose:
+        files:
+            - stdout
```

Comment:
I think it's worth mentioning that, when defining an output type (table-verbose|table|json|...), the `files:` key must also be defined, even if it's just for `stdout`. It is mandatory for the parser.

What I mean is that this will have no effect, running with default behaviour (table to stdout):

```yaml
output:
    json:
```

```yaml
output:
    json:
       files:
```

But this will set output to stdout with json format:

```yaml
output:
    json:
        files:
            - stdout
```